src/api/db/models/Image.ts

```diff
@@ -124,7 +124,7 @@ export class ImageModel {
           hasNext: false,
         } as AggregationOutput<ImageSchema>;
       }
-      return await MongoPaging.aggregate(Image.collection, {
```

Comment:
This is what my lsp is saying.

<img width="1116" alt="Screenshot 2024-10-08 at 22 17 48" src="https://github.com/user-attachments/assets/9ef657dd-e0dd-4446-9b01-25f07a28d950">

Here's the source code.  It seems weird that it would be synchronous but the types.d.ts file isn't declaring it async.

```javascript
// src/@types/mongo-cursor-pagination.d.ts

  const defaultExport: {
    mongoosePlugin: any;
    aggregate: <T extends Model>(
      model: T['schema'],
      input: AggregationInput,
    ) => AggregationOutput<T>;
  };
```

Reply:
Ah yeah I think that `defaultExport.aggregate` return type needs to be wrapped in a `Promise` type. mongo-cursor-pagination [doesn't include](https://github.com/mixmaxhq/mongo-cursor-pagination/issues/51) Typescript types so we wrote these ones ourselves and this must have been an oversight on our part. I think the following should resolve that warning:

```javascript
  const defaultExport: {
    mongoosePlugin: any;
    aggregate: <T extends Model>(
      model: T['schema'],
      input: AggregationInput,
    ) => Promise<AggregationOutput<T>>;
  };
```